homeassistant/components/kodi/media_player.py

```diff
@@ -432,7 +442,11 @@ async def async_update(self):
             self._reset_state()
             return
 
-        self._players = await self._kodi.get_players()
+        try:
+            self._players = await self._kodi.get_players()
+        except jsonrpc_base.jsonrpc.TransportError:
+            self._reset_state()
+            return
```

Comment:
I'm not sure about this state. This would potentially mean that some transient error would make it seem off.
Might not be the end of the world since it'll update in 10 seconds, but I'm not sure what's the value in this.
Also might make more sense to wrap this in @cmd to be more consistent with the other methods that are communicating with Kodi.

Reply:
To be honest, this is a part of the work of @alexanv1 which I did not look into that much (I mostly focused on preventing the integration from blocking startup).
What do you suggest here? Something like this? (untested)
```suggestion
        self._players = await cmd(self._kodi.get_players())
```

Reply:
I'm suggesting adding a @cmd decorator to the method.

Reply:
I see what you mean. There are multiple calls to the kodi API in this function, this way we'll catch an exception on any of them. I will make the change.